instrumentation/play/play-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/ActionInstrumentation.java

```diff
@@ -8,6 +8,7 @@
 import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
 import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.implementsInterface;
 import static io.opentelemetry.javaagent.instrumentation.play.v2_4.PlayTracer.tracer;
+import static io.opentelemetry.javaagent.instrumentation.play.v2_4.PlaySingletons.instrumentor;
```

Comment:
hi @shivanshu1333! it looks like `PlaySingletons` is missing from the PR

Reply:
@shivanshu1333 can add this file please?

Reply:
@shivanshu1333 would you object if someone else picks up this PR from where you left off? We would like to close out the last Instrumenter API conversions soon. Thx!